test/metabase/notification/send_test.clj

```diff
@@ -549,7 +549,7 @@
 
       (is (= [high-priority middle-priority low-priority]
              (for [_ (range 3)]
-               (#'notification.send/take-notification! queue)))))))
+               (#'notification.send/take-notification-with-timeout! queue 1000)))))))
```

Comment:
What do you think about making a (non-prototype) `take-notification!` method with a default timeout?

Reply:
he he yea, a bit of a vim curse because I can do this edit so fast I often don't think about making it shorter.